x-pack/test/functional/page_objects/api_keys_page.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export function ApiKeysPageProvider({ getService }: FtrProviderContext) {
+  const testSubjects = getService('testSubjects');
+
+  return {
+    async noAPIKeysHeading() {
+      return await testSubjects.find('NoApiKeysHeader');
+    },
+    async goToConsoleButton() {
```

Comment:
nit: this function name makes me think we are going to navigate to the console...button.
What do you think about something like `getGoToConsoleButton()`?